prbt_default_application/launch/my_application.launch

```diff
@@ -59,9 +59,15 @@ limitations under the License.
 
       <!-- Safe-torque-off-input from the safety controller is read to brake the robot before power-off
            The signal is read via modbus and you can configure the modbus server ip -->
-      <!-- <arg name="sto" value="pnoz" /> -->
+      <!-- <arg name="sto" value="pss4000" /> -->
       <!-- <arg name="sto_modbus_server_ip" value="192.168.0.10" /> -->
 
+      <!-- You can additionally select the two safety functions below: -->
+      <!-- Set braketest usage -->
+      <arg name="has_braketest_support" default="true"/>
```

Comment:
Are braketest and operation mode enabled on purpose? "You can additionally" sounds like you would let the user choose, if he wants to enable this feature? Selectable features are commented in the template to let the user change the default value by uncommenting the line(s).
```suggestion
      <!-- <arg name="has_braketest_support" default="true"/> -->
```

Reply:
Yes. But I think the whole setup including control cabinet etc is going to be our default setup, right? So I change the wording in the comment, ok?

Reply:
Or change the default to true in robot.launch (if that setup is the main use case) and leave a commented `false` in here to disable it...